TrackerConditions/fcl/prolog.fcl

```diff
@@ -190,6 +190,8 @@ StrawResponse : {
     0.0705138 , 0.0657311 , 0.0559622 , 0.0685536 , 0.0604229 , 0.0574693 ,
```

Comment:
Please add a comment documenting the provenance of these numbers

Reply:
These are pretty old, I think from sims a few years ago, the new values are in the Production PR